types/config/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for node-config
+// Type definitions for node-config 3.3
```

Comment:
In light of this change, could you also try to delete the `"dt-header": false` line in the tslint.json file? Here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/76ce0f70301c10288e823560bcadcabb27283bf2/types/config/tslint.json#L4
This would help us automatically check if the version provided here matches an existing version on npm. Thanks!

Reply:
Good idea, done in b79b4bb59a